docs/authentication/configuration/sign-up-sign-in-options.mdx

```diff
@@ -135,35 +139,35 @@ Clerk offers three verification methods:
 - **Email verification code**
 - **SMS verification code**
 
-Each of these verification methods work in the same way as their authentication strategy counterparts, but are used to verify a user's identifier rather than authenticate a user. For example, when a user adds an email address to their profile, they can receive an **Email verification link** or **Email verification code** to verify the new email address.
+These methods work similarly to their authentication strategy counterparts but are used for verifying identifiers rather than authentication. For example, when a user adds an email address to their profile, they can receive an **Email verification link** or **Email verification code** to verify the new email address.
 
 To configure verification methods:
 
 1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) and select your application.
 1. In the navigation sidebar, select **User & Authentication > Email, Phone, Username**.
-1. Select the settings cog icon next to the identifier, such as **Email address** or **Phone number**, to open the configuration settings.
-1. Under the **Verification methods** section, toggle on the verification methods you would like to enable.
-1. Select **Continue** to save your changes.
+1. Select the toggle switch icon next to the identifier, such as **Email address** or **Phone number**, to open the configuration settings.
+1. In the **Authentication strategies** section, enable the toggle for the authentication methods you would like to use.
+1. Select **Save** to save your changes.
 
 ## Social connections (OAuth)
 
-Clerk offers a number of [social login providers](/docs/authentication/social-connections/overview) that can be used during sign up and sign in. What makes this authentication option appealing is that users often won't need to enter additional contact information since the provider already has it.
+Clerk offers several [social login providers](/docs/authentication/social-connections/overview) for use during sign-up and sign-in. This authentication option is appealing because users often don't need to enter additional contact information since the provider already has it.
 
-Clerk has designed the social login process to be as seamless as possible. If an existing user tries to sign up with a social provider, the social login process is smart enough to automatically convert the sign up process into a sign in flow. The reverse is also true: if a user signs in with a social provider, but doesn't have an account yet, Clerk will automatically create one for them.
+Clerk's social login process is designed to be seamless. If an existing user attempts to sign up with a social provider, the system automatically switches to sign-in. Similarly, if a user tries to sign in with a social provider but doesn't have an account, Clerk will automatically create one.
```

Comment:
let's say OAuth instead of social login - because its not just logging in, its also signing up.

```suggestion
Clerk's OAuth process is designed to be seamless. If an existing user attempts to sign up with a social provider, the system automatically switches to sign-in. Similarly, if a user tries to sign in with a social provider but doesn't have an account, Clerk will automatically create one.
```